src/creature.js

```diff
@@ -1620,10 +1620,6 @@ export class Creature {
 
 		// As hex occupation changes, path must be recalculated for the current creature not the dying one
 		game.activeCreature.queryMove();
-
-		// Queue cleaning
-		game.UI.updateActivebox();
-		game.UI.updateQueueDisplay(); // Just in case
```

Comment:
@russon77 These methods, I was on mobile earlier it wouldnt let me comment here. If we know where they are being called that negates the needs for these 2 lines, it can possibly help optimize other similar related areas in the future.

Reply:
`updateActivebox` is called at the start of a creature's turn. it causes the fall-down animation.

the ability state is handled elsewhere.